index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Porfolio</title>
+    <script src="https://kit.fontawesome.com/d6cd3c7604.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <!--HERO SECTION -->
+    <header class="nav-container">
+        <div class="logo">
+            My logo
+        </div>
+        <nav class="toolbar">
+            <div class="nav-link">Portfolio</div>
+            <div class="nav-link">About</div>
+            <div class="nav-link">Contact</div>    
+            </nav>
+
+    </header>
```

Comment:
kindly according to the template you have choose the navbar should contain the hamburger like [here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m1_setup/toolbar-mobile.png)